stemcell_builder/stages/bosh_azure_agent_settings/apply.sh

```diff
@@ -16,24 +16,14 @@ cat > $chroot/var/vcap/bosh/agent.json <<JSON
   "Infrastructure": {
     "Settings": {
       "Sources": [
-        {
-          "Type": "ConfigDrive",
-          "DiskPaths": [
-            "/dev/disk/by-label/azure_cfg_dsk",
-            "/dev/disk/by-label/AZURE_CFG_DSK"
-          ],
-          "MetaDataPath": "configs/MetaData",
-          "UserDataPath": "configs/UserData"
-        },
```

Comment:
There does still seem to be code in the azure cpi for [ConfigDrive](https://github.com/cloudfoundry/bosh-azure-cpi-release/blob/master/src/bosh_azure_cpi/lib/cloud/azure/disk/config_disk_manager.rb), but from what we can tell, it was added by Microsoft back in 2018, then 2 months later the `spec` property was removed, so you've never really been able to use a ConfigDrive. We're not really sure why it was added, but it seems to never be used, so this removal should be safe (and Jammy seems like a good chance to rip it out)